interface/src/main/java/xsbti/Logger.java

```diff
@@ -5,9 +5,9 @@
 
 public interface Logger
 {
-	public void error(F0<String> msg);
-	public void warn(F0<String> msg);
-	public void info(F0<String> msg);
-	public void debug(F0<String> msg);
-	public void trace(F0<Throwable> exception);
+	void error(F0<String> msg);
```

Comment:
Found this on StackOverflow, JLS says http://docs.oracle.com/javase/specs/jls/se7/html/jls-9.html#jls-9.4

> It is permitted, but discouraged as a matter of style, to redundantly specify the public and/or abstract modifier for a method declared in an interface.